tests/insert_reference/target.html.expected

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>Online documentation</title>
+    </head>
+    <body>
+      <h1> Ocaml Documenation </h1>
+      <p>
+        <!-- dis::source.ml::3::1 -->
```

Comment:
Why do you want to use two colons instead of one?
Also, I would prefer "distar" instead of "dis".

Reply:
I was thinking of two colons instead of one to avoid error with names that contain one colon...
Ex : "my:source.ml"

Reply:
and what about filenames containing two colons?

You can ask filenames to be double-quoted, using standard escaping sequences to deal with occurrences of the symbol `"`.